clang/include/clang/Tooling/DependencyScanning/ModuleDepCollector.h

```diff
@@ -308,6 +308,9 @@ class ModuleDepCollector final : public DependencyCollector {
                                 ModuleDeps &Deps);
 };
 
+/// Resets some options that introduce dependencies unnecessarily.
+void removeUnnecessaryDependencies(CompilerInvocation &CI, bool ForModuleBuild);
```

Comment:
```suggestion
void removeUnnecessaryFileArguments(CompilerInvocation &CI, bool ForModuleBuild);
```

Dependencies is a bit ambiguous here.

Reply:
New name in updated commit.